images/upload-gitlab-failure-logs/taxonomy.yaml

```
@@ -2,6 +2,21 @@
 taxonomy:
   version: "0.0.0-dev"
   error_classes:
+    network_timeout:
+      grep_for:
+        - "socket.timeout"
+        - "curl: (28)"
+
+    network_error:
+      grep_for:
+        - "curl: (22)"
+        - "HTTP Error 404"
+        - "curl: (60)"
+
+    concretization_error:
+      grep_for:
+        - "does not satisfy"
```

I found another string in logs from last week
```suggestion
        - "does not satisfy"
        - "Error: errors occurred during concretization"
```

---

https://gitlab.spack.io/spack/spack/-/jobs/6342418